UPGRADE_GUIDE.md

```diff
@@ -2,22 +2,130 @@
 
 Use the following guide to assist in the upgrade process of the `easypost-java` library between major versions.
 
-* [Upgrading from 4.x to 5.0](#upgrading-from-4x-to-50)
+- [Upgrading from 5.x to 6.0](#upgrading-from-5x-to-60)
+- [Upgrading from 4.x to 5.0](#upgrading-from-4x-to-50)
+
+## Upgrading from 5.x to 6.0
+
+### 6.0 High Impact Changes
+
+- [Client Instance](#60-client-instance)
```

Comment:
Question: Does `Client Object` make more sense here? I'm not married to either but a thought?

Reply:
I was following the C# upgrade guide to be consistent. What if we name the title to `Thread-safe`, I think this is more sense than `Client Instance/ Object`?

Reply:
Ok, let's leave it as `Client Instance`